specification/containerservices/resource-manager/Microsoft.ContainerService/stable/2018-03-31/managedClusters.json

```diff
@@ -0,0 +1,1290 @@
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI of the storage account where diagnostics are stored."
+        }
+      },
+      "description": "Profile for diagnostics on the container service VMs.",
+      "required": [
+        "enabled"
+      ]
+    },
+    "ManagedClusterListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagedCluster"
+          },
+          "description": "The list of managed clusters."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of managed cluster results.",
+          "readOnly": true
+        }
+      },
+      "description": "The response from the List Managed Clusters operation."
+    },
+    "ManagedCluster": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        },
+        {
+          "properties": {
+            "properties": {
+              "description": "Properties of a managed cluster.",
+              "$ref": "#/definitions/ManagedClusterProperties",
+              "x-ms-client-flatten": true
+            }
+          }
+        }
+      ],
+      "description": "Managed cluster."
+    },
+    "ManagedClusterProperties": {
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The current deployment or provisioning state, which only appears in the response."
+        },
+        "kubernetesVersion": {
+          "type": "string",
+          "description": "Version of Kubernetes specified when creating the managed cluster."
+        },
+        "dnsPrefix": {
+          "type": "string",
+          "description": "DNS prefix specified when creating the managed cluster."
+        },
+        "fqdn": {
+          "readOnly": true,
+          "type": "string",
+          "description": "FDQN for the master pool."
+        },
+        "agentPoolProfiles": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagedClusterAgentPoolProfile"
+          },
+          "description": "Properties of the agent pool."
+        },
+        "linuxProfile": {
+          "$ref": "#/definitions/ContainerServiceLinuxProfile",
+          "description": "Profile for Linux VMs in the container service cluster."
+        },
+        "servicePrincipalProfile": {
+          "$ref": "#/definitions/ContainerServiceServicePrincipalProfile",
+          "description": "Information about a service principal identity for the cluster to use for manipulating Azure APIs. Either secret or keyVaultSecretRef must be specified."
+        },
+        "addonProfiles": {
+          "additionalProperties": {
+            "$ref": "#/definitions/ManagedClusterAddonProfile"
+          },
+          "description": "Profile of managed cluster add-on."
+        },
+        "enableRBAC": {
```

Comment:
looks good, simple and accurate.

Reply:
Just so you know, `autorest` complains about this field not being proper CamelCase. (I guess it expected `enableRbac`.) Since it's in acs-engine and the RP, I don't think it's reasonable to change it now.